packages/backend/src/index.ts

```diff
@@ -108,13 +109,17 @@ export function initIocContainer(
       replica_addresses: config.tigerbeetleReplicaAddresses
     })
   })
-  container.singleton('openApi', async (deps) => {
-    const config = await deps.use('config')
-    return await createOpenAPI(config.openPaymentsSpec)
-  })
-  container.singleton('authOpenApi', async (deps) => {
-    const config = await deps.use('config')
-    return await createOpenAPI(config.authServerSpec)
+  container.singleton('openApi', async () => {
+    const authServerSpec = await createOpenAPI(
+      path.resolve(__dirname, './openapi/auth-server.yaml')
+    )
+    const resourceServerSpec = await createOpenAPI(
+      path.resolve(__dirname, './openapi/resource-server.yaml')
+    )
```

Comment:
Could we add a good error message here, for when the files can't be found?

https://interledgerfoundation.slack.com/archives/C0302LMKBT7/p1670413450794209?thread_ts=1670407497.730339&cid=C0302LMKBT7

Reply:
As long we remember not to change the command :sweat_smile: